test/src/main/java/org/apache/accumulo/test/metrics/MetricsIT.java

```diff
@@ -287,4 +287,28 @@ public void metricTags() throws Exception {
           });
     }
   }
+
+  @Test
+  public void fateMetrics() throws Exception {
+    doWorkToGenerateMetrics();
+    cluster.stop();
+
+    List<String> statsDMetrics;
+
+    while (!(statsDMetrics = sink.getLines()).isEmpty()) {
+      statsDMetrics.stream().filter(line -> line.startsWith("accumulo.fate.tx"))
+          .map(TestStatsDSink::parseStatsDMetric).forEach(a -> {
+            var t = a.getTags();
+            log.debug("METRICS, received from statsd - name: '{}' num tags: {}, tags: {} = {}",
+                a.getName(), t.size(), t, a.getValue());
+
+            // Verify the fate metrics contain state and instanceType
+            // Checking the value would be hard to test because the metrics are updated on a timer
+            // and fate transactions get cleaned up when finished so the current state is a bit
+            // non-deterministic
+            assertNotNull(a.getTags().get("state"));
+            assertNotNull(a.getTags().get("instanceType"));
```

Comment:
Could these verify the values of the tags fall within the known set of valid values?

Reply:
Yeah I can add that for sure